include/avif/avif.h

```diff
@@ -602,9 +578,26 @@ AVIF_API uint32_t avifRGBImagePixelSize(const avifRGBImage * rgb);
 AVIF_API void avifRGBImageAllocatePixels(avifRGBImage * rgb);
 AVIF_API void avifRGBImageFreePixels(avifRGBImage * rgb);
 
-// The main conversion functions
-AVIF_API avifResult avifImageRGBToYUV(avifImage * image, const avifRGBImage * rgb);
-AVIF_API avifResult avifImageYUVToRGB(const avifImage * image, avifRGBImage * rgb);
+// The main conversion options.
+typedef enum avifConversionFlag
+{
+    // Conversion between RGB and YUV samples:
+    AVIF_CONVERSION_AUTO = 0,                // prefers libyuv if available (default)
+    AVIF_CONVERSION_AVOID_LIBYUV = (1 << 0), // only picks built-in paths
```

Comment:
Change the comment to "Only picks built-in paths. The default is to prefer libyuv if available."